adders/paraglide/config/adder.ts

```diff
@@ -0,0 +1,250 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { defineAdderConfig } from '@svelte-add/core';
+import { options, parseLanguageTagInput } from './options';
+import { HtmlElement, HtmlElementType } from '../../../packages/ast-tooling';
+
+const DEFAULT_INLANG_PROJECT = {
+	$schema: 'https://inlang.com/schema/project-settings',
+	// sourceLanguageTag: 'en',
+	// languageTags: ['en'],
+	modules: [
+		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern@latest/dist/index.js',
```

Comment:
I'm not sure I love the idea of dynamically fetching and running code. Could these be installed in `package.json`? Though it would be a lot of packages to install... It would be nice if they all lived in a single package

Reply:
@benmccann @manuel3108 would storing plugins directly in an inlang folder be better? 

Here is a 3 min loom explainer https://www.loom.com/share/ebb2f7e988744eb4959b1e0965e69f7d

```
.
└── project.inlang/
    ├── plugins/
    │   ├── i18next.js
    │   ├── m-function-matcher.js
    │   └── inlang-message-format.js
    └── settings.json
```

Reply:
Perhaps what might be nice to do here would be to change https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern@latest/dist/index.js to https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern@1.4.8/dist/index.js. Using `latest` means that you could get a different version each time you run it, which introduces some risk that your project could break due to a change in a plugin and it'd be quite hard to diagnose. If you manually upgraded the plugin and things broke you'd know what caused the error. This is why we use a lockfile with node modules. Losing that confidence is what caught my eye here.

Storing plugins directly in an inlang folder could also be okay. I like the idea of git controlling what version you use. It seems a little harder to update the version that you're using though. E.g. instead of just bumping a version number here, we'd have to download and install a tool to help us get a new version or would have to manually construct the URL to get it from npm and then copy and paste it in. The thing I do like about storing it in git is that it greatly reduces being exposed to the risk of jsdelivr being compromised and also reduces the risk of being impacted by a temporary outage. But I'm not sure I'd go that route as the developer experience doesn't seem as nice.

I think if we could find a way to support npm that would be nicest. Of course that wouldn't work on many of the platforms you support like android/ios, but perhaps it could be a node plugin to load the files from there or the CDN could be used as a fallback on those other platforms where `node_modules` would be available.

Reply:
> Losing that confidence is what caught my eye here.

Then let's pin the version. It seems like a good compromise between "building node dependencies would not work in the web and other platforms" and the desire to have reproducible builds.  

@LorisSigrist agree?

Reply:
Yes, pinning the version is a good compromise until we have a solution that works without a CDN